release/models/system/openconfig-system-logging.yang

```diff
@@ -68,12 +68,6 @@ module openconfig-system-logging {
       "IETF RFC 5424 - The Syslog Protocol";
   }
 
-  identity ALL {
```

Comment:
sorry, I did not quite understand the rationale for removing this -- how does having this identity cause issues for re-use outside of match?  (for something like listing available facilities?)

Reply:
`ALL` is not an actual facility.  Identities should be grouped in such a fashion for multiple uses and if we compare this to the use of unions throughout other models, there is often a need to choose between a precise data-type/identity *or* a catch-all like `ANY` or `ALL` which will have special implementation behavior.  While it might seem logical to group something as such, it would be better to take the union approach and keep these identities specific to the syslog facilities defined per RFC.

This is related to the `facility-override` knob included in this patch.  This provides the ability to override all facility levels towards a `remote-server` which can reuse these identities however `ALL` is not an actual identity to "set"